governance/votes/Executive vote - January 24, 2024.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Template - [Executive Vote] Stability Fee Changes, PSM-PAX-A Debt Ceiling Reduction, RETH-A Offboarding Parameters Finalization, SBE Parameter Changes, HV Bank DAO Resolution, Q4 2023 AVC Member Compensation, Spark Proxy Spell - January 24, 2024
+summary: Stability fee changes, PSM-PAX-A debt ceiling reduction and DC-IAM disabling, RETH-A offboarding parameters finalization, Smart Burn Engine parameter changes, HV Bank (RWA009-A) DAO resolution approval, AVC members compensation for Q4 2023, Spark proxy spell execution.
+address: "$spell_address"
+
+---
+# [Executive Proposal] Stability Fee Changes, PSM-PAX-A Debt Ceiling Reduction, RETH-A Offboarding Parameters Finalization, SBE Parameter Changes, HV Bank DAO Resolution, Q4 2023 AVC Member Compensation, Spark Proxy Spell - January 24, 2024
+
+The Governance Facilitators, Sidestream, Dewiz, and Phoenix Labs have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following actions will occur within the Maker Protocol:
+
+- ETH-A, ETH-B, ETH-C, WSTETH-A, WSTETH-B, WBTC-A, WBTC-B, WBTC-C [Stability Fees](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee) will be increased, as detailed below.
+- PSM-PAX-A [Debt Ceiling (`line`)](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling) will be set to zero and the vault will be removed from the [Debt Ceiling Instant Access Module (AutoLine)](https://manual.makerdao.com/module-index/module-dciam).
+- RETH-A offboarding will be finalized by changing some of its vault parameters, as detailed below.
+- SBE parameters [`bump` (lot size)](https://mips.makerdao.com/mips/details/MIP104#9-1-3-3) and [`hop` (frequency)](https://mips.makerdao.com/mips/details/MIP104#9-1-3-1) will be changed, as detailed below.
```

Comment:
It may be better to use the full name for the first time the full text references the SBE

```suggestion
- Smart Burn Engine parameters [`bump` (lot size)](https://mips.makerdao.com/mips/details/MIP104#9-1-3-3) and [`hop` (frequency)](https://mips.makerdao.com/mips/details/MIP104#9-1-3-1) will be changed, as detailed below.
```